### PR TITLE
[DropdownMenu] Update docs

### PR DESCRIPTION
**Pull request by andy-hook**

### CODE REVIEW

File: data/primitives/components/dropdown-menu/0.0.20.mdx [hunk 1]
```diff
@@ -490,6 +512,46 @@ The component that contains the dropdown menu items.
   ]}
 />
 
+### TriggerItem
+
+An item that opens a submenu. Used in combination with a nested `DropdownMenu`. Must be rendered inside `DropdownMenu.Root`.
```
Comment (benoitgrelard) [inline data/primitives/components/dropdown-menu/0.0.20.mdx:517]: That seems odd, it shouldn't be in the root right?

**Review by benoitgrelard — COMMENTED** [review]
- We should update the anatomy section too to reflect show how nested menus appear.
- I think we need to the same type changes for `onPointerDownOutside`/`onFocusOutside`/`onInteractOutside` as I've done in #180 for example
- Can we add a slight negative `alignOffset` on submenus to make sure parent/child items are aligned like in macOS?

image

File: data/primitives/components/dropdown-menu/0.0.20.mdx (hunk 1 above)
Comment (andy-hook) [reply data/primitives/components/dropdown-menu/0.0.20.mdx:517]: I think that's right, it's basically in place of a `Trigger` which is rendered directly inside of `root`, usually as the first child right?

Comment (benoitgrelard) [reply data/primitives/components/dropdown-menu/0.0.20.mdx:534]: Ah yes of course, my bad!

**Review by benoitgrelard — APPROVED** [review]

**Pull request merged**